lightning/backend/local.go

```diff
@@ -257,7 +257,7 @@ func (local *local) ShouldPostProcess() bool {
 
 func (local *local) openEngineDB(engineUUID uuid.UUID, readOnly bool) (*pebble.DB, error) {
 	opt := &pebble.Options{
-		MemTableSize:             128 << 20,
+		MemTableSize:             512 << 20,
```

Comment:
should we give this value by current memory size?

Reply:
In my benchmark for 1TB data import, this change doesn't impact much for memory usage(it will cause memory usage increase faster, but peak memory isn't increase much), and do have some performance improve. So I think this is ok. 
And if we want to set it based on system memory size, we should do more benchmarks to see the impact in different configs.

Reply:
OK, maybe leave a TODO